docs/content/workflows/design/hooks/webhook.md

```diff
@@ -1,7 +1,9 @@
 +++
 title = "Webhook"
-weight = 3
+weight = 2
 
 +++
 
-TODO
\ No newline at end of file
+On a Root Pipeline, you can add a "Webhook". Click on the created icon to get the WebHook URL.
+
```

Comment:
In order to trigger this one you just have to make a curl on the given url with the selected method